pages/services/data-science-engine/1.0.0/examples/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: layout.pug
+navigationTitle: Examples
+excerpt: Usage examples
+title: Install
+menuWeight: 7
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+This section contains usage examples for using {{ model.techName }}.
+
+# Basic
+
+Perform a default installation by following the instructions in the [Install and Customize](/services/data-science-engine/1.0.0/install/) section.
+
+## Run a {{ model.packageName }} job
+
+```bash
+dcos {{ model.packageName }} run --submit-args="--class org.apache.{{ model.packageName }}.examples.{{ model.packageName }}Pi https://downloads.mesosphere.com/{{ model.packageName }}/assets/{{ model.packageName }}-examples_2.11-2.4.0.jar 30"
+```
+
+## Run a Python {{ model.packageName }} job
+
+```bash
+dcos {{ model.packageName }} run --submit-args="https://downloads.mesosphere.com/{{ model.packageName }}/examples/pi.py 30"
+```
+
+## Run an R {{ model.packageName }} job
+
+```bash
```

Comment:
R example: https://github.com/mesosphere/jupyter-service/blob/master/tests/notebooks/r_example.ipynb

Reply:
Got it. Thanks!